pkg/local_object_storage/metabase/get.go

```diff
@@ -108,32 +109,48 @@ func getFromBucket(tx *bbolt.Tx, name, key []byte) []byte {
 	return bkt.Get(key)
 }
 
-func getVirtualObject(tx *bbolt.Tx, addr oid.Address) (*objectSDK.Object, error) {
-	bucketName := make([]byte, bucketKeySize)
-	parentBucket := tx.Bucket(parentBucketName(cnr, bucketName))
-	if parentBucket == nil {
-		return nil, logicerr.Wrap(apistatus.ObjectNotFound{})
+func getChildForParent(tx *bbolt.Tx, cnr cid.ID, parentID oid.ID, bucketName []byte) oid.ID {
+	bucketName[0] = metadataPrefix
+	copy(bucketName[1:], cnr[:])
+
+	var (
+		childOID   oid.ID
+		metaBucket = tx.Bucket(bucketName)
+	)
+	if metaBucket == nil {
+		return childOID
 	}
 
-	relativeLst, err := decodeList(parentBucket.Get(key))
-	if err != nil {
-		return nil, err
+	var (
+		parentPrefix = make([]byte, 1+len(objectSDK.FilterParentID)+1+len(parentID)+1)
+		cur          = metaBucket.Cursor()
+	)
+	parentPrefix[0] = metaPrefixAttrIDPlain
+	off := 1 + copy(parentPrefix[1:], objectSDK.FilterParentID)
+	off += copy(parentPrefix[off:], objectcore.MetaAttributeDelimiter)
+	copy(parentPrefix[off:], parentID[:])
```

Comment:
one more delim? Although resulting value is correct. `slices.Concat` as an option

Reply:
Some components are strings, some are bytes and some are not-really-slices, `slices.Concat` is not very helpful for the mix.